ingestion_service/producer/core/routers/api_endpoints_input.py

```diff
@@ -212,14 +232,18 @@ async def ingest_kg_file(
             status_code=400,
             detail="Unsupported file extension. Supported extensions: TTL and JSONLD"
         )
+    named_graph_iri = named_graph_exists(graph)
+    if named_graph_iri["status"] != True:
```

Comment:
What are examples of what the "status" value could be? Is it purely `True` or `False`? Does that represent more of if the graph "is_valid" or "is_not_valid"?

Just trying to understand key naming here

As an aside you could just so something similar to `if not named_graph_iri["status"]`